cmd/livepeer/livepeer.go

```diff
@@ -137,6 +137,7 @@ func parseLivepeerConfig() starter.LivepeerConfig {
 	cfg.MaxPricePerUnit = flag.String("maxPricePerUnit", *cfg.MaxPricePerUnit, "The maximum transcoding price per 'pixelsPerUnit' a broadcaster is willing to accept. If not set explicitly, broadcaster is willing to accept ANY price. Can be specified in wei or a custom currency in the format <price><currency> (e.g. 0.50USD). When using a custom currency, a corresponding price feed must be configured with -priceFeedAddr")
 	cfg.IgnoreMaxPriceIfNeeded = flag.Bool("ignoreMaxPriceIfNeeded", *cfg.IgnoreMaxPriceIfNeeded, "Set to true to allow exceeding max price condition if there is no O that meets this requirement")
 	cfg.MinPerfScore = flag.Float64("minPerfScore", *cfg.MinPerfScore, "The minimum orchestrator's performance score a broadcaster is willing to accept")
+	cfg.DiscoveryTimeout = flag.Int("discoveryTimeout", *cfg.DiscoveryTimeout, "Time in milliseconds to wait for first round of GetOrchestrator responses (default = 500ms)")
```

Comment:
We should use a `flag.Duration` here which renders a nicer usage experience with clear units. One can set for example `500ms` or `5s`.

If you stay with the `Int` flag, could you please rename (at least the public arg) to `discoveryTimeoutMs`? Just so the unit is clear as well.